rfcs/SemanticNullability.md

```diff
@@ -497,10 +497,10 @@ day-to-day work.
     input and output (the difference between semantic an strict non-null does
     not occur on input)
 - [F][criteria-f]
-  - ✅ `Int` reatains its meaning across both modes, and `Int!` means
-    non-nullable in both modes. Only the SDL ever uses `Int!!` and it still
-    means non-null, just with the additional "kills parent on exception"
-    behavior.
+  - 🚫 `Int` reatains its meaning across both modes, but `Int!` means
+    non-nullable in one mode, but semantic non-nullable in another. Only
```

Comment:
"modes" here was referring to null bubbling enabled/disabled, and `Int!` always means semantic non-null when null bubbling is disabled, so all solutions would fail this criteria the same. As such, the commentary around `Int!` meaning has been removed from the criteria.